thunderbird_privacy_policy/en-US.md

```diff
@@ -63,7 +63,7 @@ __Technical data for updates__: Thunderbird periodically connects to our server
 
 __Mozilla Affiliates__: Thunderbird is a project of MZLA Technologies Corporation, a subsidiary of Mozilla Foundation and as such, shares some of the same infrastructure. This means that, from time to time, your data (e.g., crash reports, and technical and interaction data) may be disclosed to Mozilla Corporation and Mozilla Foundation. If so, it will be maintained in accordance with the commitments we make in this Privacy Notice.
 
-__DNS servers, Standard Autoconfiguration URIs, and Mozilla's Configuration Database__: To simplify the email set-up process, Thunderbird tries to determine the correct settings for your account by contacting Mozilla’s configuration database as well as external servers. These include DNS servers and standard autoconfiguration URIs. During this process, Thunderbird may send your email domain to these servers, but your full email address and password are never disclosed.
+__DNS servers, Standard Autoconfiguration URIs, and Mozilla's Configuration Database__: To simplify the email set-up process, Thunderbird tries to determine the correct settings for your account by contacting Mozilla’s configuration database as well as external servers. These include DNS servers and standard autoconfiguration URIs. During this process your domain will be sent to to Mozilla's configuration database. Your email may be disclosed to your network administrators. Your email address and password may be disclosed to administrators of your domain.
```

Comment:
- Add a comma between process and you so it looks like this: `... URIs. During this process, your domain will...`
- Remove an extra "to" in `your domain will be sent to Mozilla's configuration database.`
- Add a "the" in the last sentence so it reads like this: `... be disclosed to the administrators of your domain.`